plugins/kafka/README.md

```diff
@@ -0,0 +1,175 @@
+#ECL Apache Kafka Plugin
+
+This is the ECL plugin to access [Apache Kafka](https://kafka.apache.org), a publish-subscribe messaging system.  ECL string data can be both published to and consumed from Apache Kafka brokers.
+
+Client access is via a third-party C++ plugin, [librdkafka](https://github.com/edenhill/librdkafka).
+
+##Installation and Dependencies
+
+[librdkafka](https://github.com/edenhill/librdkafka) is included as a git submodule in HPCC-Platform.  It will be built and integrated automatically when you build the HPCC-Platform project.
+
+The recommended method for obtaining Apache Kafka is via [source code](https://kafka.apache.org/documentation.html#quickstart), which also contains excellent instructions for setting up and interacting with topics.  Note that Apache Kafka has its own set of dependencies, most notably [zookeeper](https://zookeeper.apache.org).
```

Comment:
It might be worth adding a link to http://kafka.apache.org/documentation.html#quickstart 
Gives the user a little extra help. Also explaining that they only need to start a zookeeper server and a kafka server to be able to use your example ecl, kafkatest.ecl, which might also be useful to link & ref in this doc. It may also be useful to explicitly state that comparison to the key might be useful to indicate whether they have setup and started the servers adequately  (something I personally didn't include in the redis readme and should do).